getting-started/templates/systemlink-secrets.yaml

```diff
@@ -38,6 +38,14 @@ secrets:
     ## This cookie is used to authenticate containers in the RabbitMQ stateful set. Use a strong random sequence of characters.
     ##
     erlangCookie: ""  # <ATTENTION>
+  # Miscellaneous artifact repository credentials.
+  genericArtifactPull:
+    # URL for the repository.
+    url: "" # <ATTENTION>
```

Comment:
How does the user learn this URL? Is it different than the global image repository? Can we not build this up from existing metadata in the chart?

Reply:
It's the same as the global image registry, but with a different subpath. We already have a global value `imageRegistry`, which would be something like `niartifacts.jfrog.io/rnd-docker-ci`. I suppose we could break that up into separate values, like:
```
artifactBase: "niartifacts.jfrog.io"
imageRegistryPath: "rnd-docker-ci"
genericArtifactPath: "rnd-generic-ci"
```
This would require changes to pretty much every helm chart, though, so I'm a little worried about the churn. What do you think?

Reply:
Seems like a lot of work to avoid setting one value. Do try to document as best we can what the user need to know. And provide a default here for the standard niedge location so that users who are not mirroring do not need to modify this value.

Also, I have a document about mirroring requirements in review: https://github.com/ni/install-systemlink-enterprise/pull/40

What needs to be done about these files?

Reply:
[This PR](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/302710) adds generic artifacts to the release bundle. So there will also be a `ni-generic` repo on the edge node. I'm not familiar with the specifics of how the pull-through cache works, but I don't see why it wouldn't work the same as with images and charts.